aries-site/src/layouts/main/Layout.js

```diff
@@ -1,27 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Grommet, Main, ResponsiveContext } from 'grommet';
+import { Box, Grommet, Main, ResponsiveContext } from 'grommet';
 
 import { aries } from '../../themes/aries';
 import { Header, Head, SidebarLayout } from '..';
 
+const calcPad = size => {
+  const val = size !== 'small' ? 'xlarge' : 'large';
+  return val;
+};
+
 export const Layout = ({ children, title, isLanding }) => {
   return (
     <Grommet theme={aries} full>
       <ResponsiveContext.Consumer>
         {size => (
-          <>
+          <Box
+            // pad={{ horizontal: calcPad(size) }}
+            width={{ max: 'xxlarge' }}
+            margin="auto"
+          >
             <Head title={title} />
             <Header showLinks={!isLanding} />
             {!isLanding ? (
               <SidebarLayout mainContentChildren={children} />
             ) : (
               // aligns with responsive padding for aries-core Nav
-              <Main pad={size !== 'small' ? 'xlarge' : 'large'}>
-                {children}
-              </Main>
+              <Main pad={calcPad(size)}>{children}</Main>
```

Comment:
```suggestion
              <Main pad={{horizontal: calcPad(size), bottom: calcPad(size), top: "medium"}}>{children}</Main>
```
The space between the Header and the landing page content feels really big. Based on Chris's designs, the ideal distance is somewhere in between a top padding of "medium" and "large", so if we had to choose one, I think that "medium" looks nicer. This is just a thought though.

Reply:
Done. we might want to circle back to it once the Nav will have changes. but I'm OK with having it in the meanwhile.